src/js/angular/repositories/fedx-repo.directive.js

```diff
@@ -218,25 +218,37 @@ function fedxRepoDirective($modal, RepositoriesRestService, toastr, $timeout) {
 
             } else if ($scope.model.repositoryName && $scope.model.store === REMOTE_REPO_STORE) {
                 member = {
-                    store : REMOTE_REPO_STORE,
-                    repositoryName : $scope.model.repositoryName,
-                    repositoryServer : $scope.model.repositoryServer,
-                    username : $scope.model.username,
-                    password : $scope.model.password,
+                    store: REMOTE_REPO_STORE,
+                    repositoryName: $scope.model.repositoryName,
+                    repositoryServer: $scope.model.repositoryServer.charAt($scope.model.repositoryServer.length - 1) === '/' ? $scope.model.repositoryServer.substring(0, $scope.model.repositoryServer.length - 1) : $scope.model.repositoryServer,
```

Comment:
You can use $scope.model.repositoryServer(-1) to get the last character and .slice(0, -1) to get a string until the last character. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice